src/engine/SCons/Tool/__init__.py

```diff
@@ -136,7 +137,7 @@ def _tool_module(self):
         sys.path = self.toolpath + sys.path
         # sys.stderr.write("Tool:%s\nPATH:%s\n"%(self.name,sys.path))
 
-        if sys.version_info[0] < 3 or (sys.version_info[0] == 3 and sys.version_info[1] in (0,1,2,3,4)):
```

Comment:
Hmm. the py3 < 3.5 code shouldn't have ever been here..

Reply:
I will gladly remove the other part of the condition, but that's what I've found in your code. Should I?

Reply:
Well, this is primarily for Py2 code.